node/src/service.rs

```diff
@@ -115,6 +122,27 @@ thread_local!(static TIMESTAMP: std::cell::RefCell<u64> = std::cell::RefCell::ne
 /// Provide a mock duration starting at 0 in millisecond for timestamp inherent.
 /// Each call will increment timestamp by slot_duration making Aura think time has passed.
 struct MockTimestampInherentDataProvider;
```

Comment:
This can be moved to node_common as well

Reply:
It depends on the `SLOT_DURATION` in the runtimes, but we can probably be generic over it.